doc/tcpwer.md

```diff
@@ -0,0 +1,37 @@
+# Time-Constrained minimum Permutation Word Error Rate
+
+The Time-Constrained minimum Permutation Word Error Rate (tcpWER) is similar to the cpWER, but uses temporal information to prevent matching words that are far apart temporally.
+The temporal constraint idea is similar (but not identical) to aslicte's `-time-prune` option.
+
+## Goals of the tcpWER
+The system should be forced to provide rough temporal annotations (diarization) and should be penalized when its results become implausible compared to the reference. 
+This leads us to following properties:
+
+- The system should group segments that it thinks belong to the same speaker together (similar to cpWER)
+- It should not be penalized when the system combines several words (e.g. an utterance) in one segment, but
+- The system should be penalized when it produces long segments spanning multiple reference segments
+- It should not be penalized when the system provides more precise timing than the reference (e.g., by splitting in a pause or producing tighter bounds)
+
+## Pseudo-word-level annotations
+To compute the matching, we need a temporal annotation (start and end time) for each word.
+Often, detailed word-level temporal annotations are not available, either because annotating a reference is expensive or a system does not produce such detailed information.
+We thus implement different strategies to infer "pseudo-word-level" timings from segment-level timings:
+
+- `full_segment`: Copy the time annotation from the segment to every word within that segment
```

Comment:
Mention, that this is good for ref and bad for hyp?

ref: Humans might group words with long pauses in between, e.g. in LibriSpeech, where ...

hyp: `full_segment` can be fooled (e.g. set start to 0 and end to a large number to get the cpWER), hence avoid it.

Have we already a suggestion, if `equidistant_intervals` or `euqidistant_points` should be used?